src/NzbDrone.Core/Datastore/Migration/212_add_role_to_users.cs

```diff
@@ -0,0 +1,14 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(212)]
```

Comment:
Another PR is already using migration 212, this should be done in the same migration as adding the API key to the user.